src/main/java/gift/config/WebConfig.java

```diff
@@ -32,4 +34,17 @@ public void addArgumentResolvers(List<HandlerMethodArgumentResolver> resolvers)
         resolvers.add(resolver);
         resolvers.add(loginUserArgumentResolver);
     }
+
+    public static final String[] ALLOWED_METHOD_NAMES = {
+            "GET", "POST", "PUT", "DELETE", "OPTIONS"
+    };
```

Comment:
하드코딩하기보다는 가능한 상수를 활용하면 실수를 방지할 수 있을 것 같아요. 😃 

```suggestion
    public static final String[] ALLOWED_METHOD_NAMES = Stream.of(HttpMethod.GET, HttpMethod.POST, HttpMethod.PUT, HttpMethod.DELETE, HttpMethod.OPTIONS)
        .map(HttpMethod::name)
        .toArray(String[]::new);
```